File: src/main/java/vergisst/minecraftmod/weaponthrow/Interface/IPlayerEntityMixin.java
package vergisst.minecraftmod.weaponthrow.Interface;

import vergisst.minecraftmod.weaponthrow.capabilities.PlayerThrowData;

public interface IPlayerEntityMixin {
    void weaponThrow$setThrowPower(PlayerThrowData value);
    PlayerThrowData weaponThrow$getThrowPower();
}

File: src/main/kotlin/vergisst/minecraftmod/weaponthrowlite/api/IPlayerEntityMixin.kt
package vergisst.minecraftmod.weaponthrowlite.api

import vergisst.minecraftmod.weaponthrowlite.capabilities.PlayerThrowData

interface IPlayerEntityMixin {
    fun setThrowPower(value: PlayerThrowData)

    fun getThrowPower(): PlayerThrowData
}